app/helpers/rules_helper.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RulesHelper
+  require "govspeak"
```

Comment:
[suggestion]
I don't think you need this line, it should be required throughout the project

Reply:
You're right. In the past I've had to specify it but it doesn't look like I need to here.

Good spot - thank you.